### PR TITLE
Quick Sort in Python

### PR DESCRIPTION
**Pull request by ranesr**
This pull request is for issue number #34

### CODE REVIEW

File: Sorting Algorithms/Quick Sort Algorithm/Python/QuickSort.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+#
+# Quick Sort Algorithm
+# http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/
+# GitHub - https://github.com/VamsiSangam/theoryofprogramming
+# Code Contributor - Saurabh Rane (github.com/ranesr)
+#
+# ===== ===== ===== */
+
+# Quick Sort Function
+def quickSort(alist):
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Python/QuickSort.py]: Name this function ``` quick_sort ```.

File: Sorting Algorithms/Quick Sort Algorithm/Python/QuickSort.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	quickSortHelper(alist, 0, len(alist) - 1)
+
+
+# Helper Function
+def quickSortHelper(alist, start, end):
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Python/QuickSort.py]: Name this function ``` _quick_sort_helper ```.

**Review by VamsiSangam — COMMENTED** [review]
We will follow the '_' naming convention for naming python methods from now. For methods meant to be public, every word is lower case separated by '_', and for a method meant to be private, it begins with '_' and follows the same rules as above. The rest of the code is fine :+1:

**Comment by ranesr** [comment]
@VamsiSangam I made changes as suggested. I also updated `def partition` to `def _partition`.

**Review by VamsiSangam — APPROVED** [review]

**Comment by VamsiSangam** [comment]
Cool! Thanks for your contribution! You should be able to see your code live on the [corresponding post](http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/) in a few minutes. :smile:

**Pull request merged**